wire-ios/Wire-iOS/Sources/UserInterface/Location/MapManager.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import MapKit
+
+// MARK: - MapManagerDelegate
+
+protocol MapManagerDelegate: AnyObject {
+    func mapManager(_ manager: MapManager, didUpdateUserLocation userLocation: MKUserLocation)
+    func mapManager(_ manager: MapManager, regionDidChangeAnimated animated: Bool)
+    func mapManagerDidFinishRenderingMap(_ manager: MapManager, fullyRendered: Bool)
+}
+
+// MARK: - MapManager
+
+class MapManager: NSObject {
+
+    // MARK: - Properties
+
+    let mapView = MKMapView()
```

Comment:
It seems a bit strange that a manager class has a reference to a view. Perhaps instead of `MapManager` it could be a `MapViewController`?

Reply:
Done!